nginx/main.conf

```diff
@@ -32,4 +32,12 @@ http {
       
     include service_wildcard.conf;
   }
+
+  server {
+    listen 8090;
+
+      location /healtz {
+        return 200;
```

Comment:
This is not very useful health check no? 


Also the name is not really useful. Is it for liveness probe? Than just opening socket would do?